README.md

```diff
@@ -482,8 +502,6 @@ https://hla.alleles.org/nomenclature/index.html
 
 ### Allele Frequency Net Database
 
-http://www.allelefrequencies.net/
```

Comment:
It seems like the final commit is lacking this link.

I agree with you that the home page seems to be down, so it's ok to change it to a different link like "About Us".

Could you put it back in?